jwst/associations/__init__.py

```diff
@@ -1,4 +1,12 @@
-"""Setup default and environment """
+"""Association Generator
+
+The Association Generator takes a list of items, an Association Pool, and
+creates sub-lists of those items depending on each items' attributes. How the
```

Comment:
I think this should be "each item's" instead of "each items'", because "each" is singular.